source/localizable/applications/initializers.md

```diff
@@ -89,5 +89,17 @@ export default {
 };
 ```
 
+```app/initializers/asset-init.js
+export function initialize(application) {
+  // ... your code ...
+};
+
+export default {
+  name: 'assetInit',
+  after: ['configReader', 'websocketInit'],
+  initialize: initialize
+};
+``
```

Comment:
Instead of adding another code block, let's just tweak the existing code block that uses `after`.  That should be clear enough.

And thanks!